goinstant/godogs_test.go

```diff
@@ -0,0 +1,175 @@
+package main
+
+import (
+	"bufio"
+	"bytes"
+	"fmt"
+	"os"
+	"os/exec"
+	"path/filepath"
+	"runtime"
+	"strings"
+	"sync"
+
+	"github.com/cucumber/godog"
+	"github.com/pkg/errors"
+)
+
+var (
+	binaryFilePath string
+	logs           string
+	directoryNames []string
+)
+
+func theCommandIsRun(command string) error {
+	res, err := runTestCommand(binaryFilePath, strings.Split(command, " ")...)
+	if err == nil {
+		logs = res
+	}
+	return nil
+}
+
+func checkTheCLIOutputIs(expectedOutput string) error {
+	err := compareStrings(logs, expectedOutput)
+	if err != nil {
+		return err
+	}
+	return nil
+}
+
+func checkCustomPackages(packages *godog.Table) error {
+	head := packages.Rows[0].Cells
+
+	for i := 1; i < len(packages.Rows); i++ {
+
+		for n, cell := range packages.Rows[i].Cells {
+			switch head[n].Value {
+			case "directory":
+				directoryNames = append(directoryNames, cell.Value)
+			case "location":
+				err := compareStrings(logs, cell.Value)
+				if err != nil {
+					return err
+				}
+			default:
+				return errors.New("Unexpected column name: " + head[n].Value)
+			}
+		}
+	}
+
+	// Cleanup downloaded packages post test
+	deleteContentAtFilePath([]string{".", "features"}, directoryNames)
+	return nil
+}
+
+func compareStrings(inputLogs, expectedOutput string) error {
```

Comment:
Can we please give this function a better name?

Reply:
On the second name already, do you have a suggestion?